src/main/java/tictactoe/Menu.java

```diff
@@ -21,6 +24,19 @@ public GameTypes getGameType() {
         return GameTypes.values()[indexGameTypeSelected];
     }
 
+    public boolean replay() {
+        io.write(REPLAY_REQUEST);
+        String answer = io.read();
+
+        if (answer.equals(YES)) {
```

Comment:
Something to think about (not necessarily for this pull request) - if the player types 'Yes' or 'yEs' - they would be reprompted. Would this be annoying for the player?   You could perform an operation on the users input to make it all lower or all upper case to eliminate re-prompting if the case doesn't match.